app/presenters/nsm/check_answers/disbursement_costs_card.rb

```diff
@@ -30,7 +30,7 @@ def header_rows
       def disbursement_rows
         disbursements.rows.map do |row|
           {
-            head_key: row[:key][:text],
+            head_key: row[:key][:text].parameterize.underscore,
```

Comment:
so `row[:key][:text]` is the translated value so not 100% sure converting it to look like a key is a good idea. Would mostly depend on where it is then used (which I am not sure of)

but this is likely to break what is shown on the some of the views